### PR TITLE
Webhook $secret_token

### PR DESCRIPTION
**Pull request by DJTommek**
- added support for `SetWebhook::$secret_token`
- added example/set-webhook.php

### CODE REVIEW

File: examples/set-webhook.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+$webhook->url = 'https://example.com/';
+$webhook->max_connections = 3;
+$webhook->allowed_updates = ['message'];
+$webhook->drop_pending_updates = true;
+$webhook->secret_token = 'some-very-secret-string';
+
+$promise = $tgLog->performApiRequest($webhook);
+$promise->then(
+    function (TelegramTypes $response) {
+        var_dump('Webhook was successfully set. Run get-webhookinfo.php to get detailed information about webhook from server.');
```
Comment (NanoSector) [inline examples/set-webhook.php:26]: This could be turned into an echo instead

**Review by NanoSector — CHANGES_REQUESTED** [review]
You add a property, but it doesn't actually do anything as of yet looking at the code, so what is the purpose of setting it like this?

**Review by NanoSector — APPROVED** [review]
Sorry - I forgot these things get serialised based on properties. LGTM then, minor comment on the example.

File: examples/set-webhook.php (hunk 1 above)
Comment (DJTommek) [reply examples/set-webhook.php]: Converted to echo, @NanoSector.

**Pull request merged**